src/main/java/Eln.java

```diff
@@ -0,0 +1,105 @@
+
+import java.util.Scanner;
+
+
+public class Eln {
+    private static final String LINE = "____________________________________________________________";
+    private static Task[] tasksList = new Task[100]; // max number of tasks set to 100
+    private static int numOfTasks = 0;
+
+    private static void greeting() {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm Eln");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE);
+    }
+
+    private static void farewell() {
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(LINE);
+    }
+
+    private static void showList() {
+        System.out.println(LINE);
+        if(numOfTasks == 0) {
+            System.out.println("there are no pending tasks");
+        }
+        for(int i = 0; i < numOfTasks; i++) {
+            System.out.println((i+1) + ". " + tasksList[i].toString());
+        }
+        System.out.println(LINE);
+    }
+
+    private static void addTask(String taskToAdd) {
+
+        Task newTask;
+
+        if (taskToAdd.startsWith("todo")) {
+            newTask = new Todo(taskToAdd);
+        } else if (taskToAdd.startsWith("deadline")) {
+            newTask = new Deadline(taskToAdd);
+        } else if (taskToAdd.startsWith("event")) {
+            newTask = new Event(taskToAdd);
+        } else {
+            newTask = new Task(taskToAdd);
+        }
```

Comment:
might want to be careful with putting this as the default option/else loop. A task can either be a todo, deadline or event - but if user inputs some other string, a (general) Task is added which does not fall in any of the task types mentioned!